lib/querystring.js

```diff
@@ -76,21 +84,21 @@ function unescapeBuffer(s, decodeSpaces) {
   var hasHex = false;
   while (index < s.length) {
     currentChar = s.charCodeAt(index);
-    if (currentChar === 43 /* '+' */ && decodeSpaces) {
-      out[outIndex++] = 32; // ' '
+    if (currentChar === CHAR_PLUS && decodeSpaces) {
```

Comment:
Understanding this code requires looking at the top of file to see what CHAR_PLUS is, but its not defined there, so then I have to look in another file to see the definition: `CHAR_PLUS: 43, /* + */`, which is exactly what used to be in this file. Its not clear to me that the levels of indirection are an improvement.

Reply:
I kind of agree, but this has been done in many other places, so I think this is good for consistency.